dev-libs/actor-framework/actor-framework-0.17.3.ebuild

```diff
@@ -0,0 +1,110 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit cmake-multilib python-single-r1
+
+DESCRIPTION="The C++ Actor Framework (CAF)"
+HOMEPAGE="https://actor-framework.org/"
+SRC_URI="https://github.com/actor-framework/actor-framework/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="|| ( Boost-1.0 BSD )"
+SLOT="0/17.3"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc examples opencl +openssl python static-libs test tools"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="examples? ( net-misc/curl[${MULTILIB_USEDEP}] )
+	openssl? ( dev-libs/openssl:0=[${MULTILIB_USEDEP},static-libs?] )
+	opencl? ( virtual/opencl[${MULTILIB_USEDEP}] )
+	python? ( ${PYTHON_DEPS}
+		dev-python/ipython
+		dev-python/pybind11[${PYTHON_USEDEP}] )"
+
+DEPEND="${RDEPEND}"
+
+BDEPEND="doc? ( app-doc/doxygen[dot]
+	app-shells/bash:0
+	dev-texlive/texlive-fontsextra
+	dev-texlive/texlive-latex
+	dev-texlive/texlive-latexextra )"
+
+RESTRICT="!test? ( test )"
+
+src_prepare() {
+	append-cflags "-std=c++11 -Wextra -Wall -pedantic"
+	append-cxxflags "-std=c++11 -Wextra -Wall -pedantic"
+
+	rm -rf libcaf_python/third_party || die
+
+	if use python; then
+		sed -i 's:.*/third_party/pybind/.*:if(FALSE):' \
+			libcaf_python/CMakeLists.txt || die
+		sed -i 's:.*pybind11/pybind11.h"$:#include "pybind11/pybind11.h":' \
+			libcaf_python/src/main.cpp || die
+	fi
+
+	cmake-utils_src_prepare
```

Comment:
By reading the multilib eclass it seems you can migrate this too from cmake-utils to cmake (https://github.com/gentoo/gentoo/blob/master/eclass/cmake-multilib.eclass).

Reply:
Done